### PR TITLE
HEL-327 HEL-330 | Update photo metadata layout

### PR DESCRIPTION
**Pull request by jannetalvio**
Update metadata header localizations. Change metadata layout in the photos page. Manipulate metadata contents by truncating fixed words out from description, era (photo taken date) and geography (address).

### CODE REVIEW

File: hkm/templatetags/hkm_tags.py [hunk 1]
```diff
@@ -125,3 +151,19 @@ def record_index(record, search_result):
...
+            result = result.split(suffix, 1)[0]
+    return result
+
+
+def trim_prefix(target, prefixes):
+    result = target
+    for prefix in prefixes:
+        if result.startswith(prefix):
+            result = result.split(prefix, 1)[1]
+    return result
```
Comment (tuomas777) [inline hkm/templatetags/hkm_tags.py:169]: Starting from Python 3.9 there are [str.removeprefix()](https://docs.python.org/3.9/library/stdtypes.html#str.removeprefix) and [str.removesuffix()](https://docs.python.org/3.9/library/stdtypes.html#str.removesuffix).

Comment (karisal-anders) [reply hkm/templatetags/hkm_tags.py:169]: I was just thinking that can we rely on at least Python 3.9 being used. README.md doesn't explicitly say that at least Python 3.9 is required but it suggests to use at least Python 3.9 for non-Docker development. At least some of the GitHub actions use Python 3.9. I don't have the kuvaselaamo docker container running at the moment so I couldn't go into it and check what Python version is installed in the docker container which is actually the point here. But barring that I think it should be ok to rely on at least Python 3.9 being installed.

Comment (tuomas777) [reply hkm/templatetags/hkm_tags.py:169]: https://github.com/City-of-Helsinki/kuvaselaamo/blob/master/Dockerfile#L2 🙂

Comment (karisal-anders) [reply hkm/templatetags/hkm_tags.py:169]: Ah! So it was there, missed that by looking at the docker-compose.yml, not the Dockerfile, and generally searching the codebase with "Python 3.". Python 3.9 it is in Docker also!

File: hkm/templatetags/hkm_tags.py [hunk 2]
```diff
@@ -19,18 +19,21 @@ def finna_image(img_id):
...
+    result = value.removeprefix("sisällön kuvaus: ")
+    result = result.removesuffix("mustavalkoinen")
+    result = result.removesuffix("värillinen") 
+    return result
 
 
 @register.filter
 def truncate_era(value):
-    return trim_prefix(value, ["kuvausaika: "])
+    return value.removeprefix("kuvausaika ")
```
Comment (karisal-anders) [inline hkm/templatetags/hkm_tags.py:30]: Did you change the removable prefix "kuvausaika: " to "kuvausaika " on purpose here?

**Review by karisal-anders — APPROVED** [review]
Approved with the upcoming truncate_era kuvausaika-prefix addition as long as all the tests pass.

**Pull request merged**